src/Microsoft.IdentityModel.Tokens/SecurityAlgorithms.cs

```diff
@@ -39,6 +39,12 @@ public static class SecurityAlgorithms
         public const string Aes256Encryption = "http://www.w3.org/2001/04/xmlenc#aes256-cbc";
         public const string DesEncryption = "http://www.w3.org/2001/04/xmlenc#des-cbc";
 
+        // Currently not supported. It's possible to provide your own implementation of the algorithm by overriding CryptoProviderFactory and AuthenticatedEncryptionProvider.
+        // See: https://www.w3.org/TR/xmlenc-core1/#sec-AES-GCM
```

Comment:
consider adding these as xml comments to all the consts. that way a developer will actually see these comments in the tooltip.

Reply:
addressed with: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1025/commits/c8e9d097cbc2e9cd1d54d0624ac67e4c6530458f